content/en/docs/contribution-guidelines/upgrade-graphs.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Channel Upgrade Graphs Legends
+linkTitle: Upgrade Graphs Legends
+description: >
+  Defines the legends for channel upgrade graph representation schemes used. 
+---
+
+By checking this document, you will be able to understand the legends used in the Channel Upgrade Graphs as how to create them. Note that it also can be useful to drawing your own graphs to represent your scenarios if you need a help with and communicate with others. 
+
+## Legends
+
+| <div style="align-text: center;width:200px">Diagram (e.g)</div> | Description |   
+|----------|:-------------:|
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::head
+{{</mermaid>}} | Operator bundle version which is the head of the channel |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::installed
+{{</mermaid>}} | Operator bundle version which is installed currently |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1)
+{{</mermaid>}} | Operator bundle version which is installable |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph preview
+      ID(v0.0.1) 
+   end
+{{</mermaid>}} | Operator bundle channel. (eg. from the bundle `annotations.yaml`, `operators.operatorframework.io.bundle.channels.v1: preview` ) |
```

Comment:
Do we need the reader of this document to have prior knowledge of what our bundle metadata is, what `annotations.yaml` is? Is "A channel versions of the operator are available in" not sufficient?